### ISSUE TITLE
superClass() on one map using previously acquired clustering

### ISSUE THREAD

**Issue by SnehaVidekar** [tuxette/SOMbrero#5]
superClass() method accepts.. somRes and also somSC object can be specified. 

I am trying to use the previously obtained clustering somSC on the new map somRes.

**Reference from SOMBrero.pdf:**

superClass(sommap, method, members, k, h, ...)

Arguments
sommap>>>> A somRes object.
method >>>>Argument passed to the hclust function.
members >>>>Argument passed to the hclust function.
k              >>>> Argument passed to the cutree function (number of super-clusters to cut the
dendrogram).
h             >>>>Argument passed to the cutree function (height where to cut the dendrogram).
                      ... Used for plot.somSC: further arguments passed either to the function plot (case
                     type="dendro") or to plot.myGrid (case type="grid") or to plot.somRes
                      (all other cases).
**x >>>> A somSC object.
object >>>> A somSC object.**

**Comment by tuxette** [comment]
Yes it is: check the 3 last plots in this vignette: https://cran.r-project.org/web/packages/SOMbrero/vignettes/c-doc-numericSOM.html
Hope that helps?

**Comment by SnehaVidekar** [comment]
lets say there is 
somRes: my.som1
somSC: sc1
somRes : my.som2

Now I want to cluster my.som2 using sc1. So from the arguments list of superClass(), I see it also has two arguments called as x and object which are somSC objects. 

trying below code
sc2 <- superClass(my.som2, k = 4, x=sc1)

or

sc2 <- superClass(my.som2, k = 4, object=sc1)

**Comment by SnehaVidekar** [comment]
Thank you for the information. 
This makes clear how the "add" option can help in different visualization. You can close this issue.
